src/components/search/SideFacetDropdown.vue

```diff
@@ -261,6 +297,17 @@
           profile: 'facets',
           facet: this.name
         };
+      },
+
+      criteria() {
+        return this.searchFacet.trim().toLowerCase();
+      },
+
+      searchOptions() {
```

Comment:
This never returns actual options.

`optionsDescription` would be a better name.

Reply:
Indeed, naming is incorrect. As the description is not even really used I've removed this prop altogether.